### PR TITLE
[jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp] fix: remove unused node handle, add "remain time" in remaining_time topic data 

### PR DESCRIPTION
**Pull request by furushchev**
- create another clean repository

```bash
git fetch origin
git checkout -b add-truck-plugin origin/master
```

- fetch files from another branch

```bash
# e.g.
git checkout tongtybj/sensor-based-control -- jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp
```

- commit and create PR

```bash
git push tongtybj add-truck-plugin
hub pull-request # or open browser and create PR which includes only minimum update of functions
```

### CODE REVIEW

File: jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp [hunk 1]
```diff
@@ -85,9 +85,6 @@ void GazeboTruck::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
   node_handle_ = new ros::NodeHandle(namespace_);
   pub_score_ = node_handle_->advertise<std_msgs::String>("score", 1, true); // set latch true
   pub_time_ = node_handle_->advertise<std_msgs::String>("remaining_time", 1);
-  ros::NodeHandle param_handle(*node_handle_, "controller");
-
-
```
Comment (k-okada) [inline jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp]: why you remove this

File: jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp [hunk 2]
```diff
@@ -165,11 +163,10 @@ void GazeboTruck::Update()
   rayShape->GetIntersection(distAbove, entityName);
   distAbove -= 0.00001;
 
-  //
```
Comment (k-okada) [inline jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp]: why you remove this

File: jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp [hunk 3]
```diff
@@ -103,6 +100,7 @@ void GazeboTruck::Update()
   if ( terminated_ ) {
     return;
   }
+
```
Comment (k-okada) [inline jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp]: why you added this

File: jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp (hunk 1 above)
Comment (tongtybj) [inline jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp]: @k-okada 

param_handle is defined here, but never be used. Is it necessary?

File: jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp (hunk 2 above)
Comment (tongtybj) [inline jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp]: @ Do we need this blank line...?

File: jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp (hunk 1 above)
Comment (k-okada) [inline jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp]: i see...

File: jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp [hunk 4]
```diff
@@ -164,12 +161,12 @@ void GazeboTruck::Update()
   rayShape->SetPoints(start, end);
   rayShape->GetIntersection(distAbove, entityName);
   distAbove -= 0.00001;
-
+
```
Comment (k-okada) [inline jsk_mbzirc_common/src/mbzirc_gazebo_truck_plugin.cpp]: do not change style

**Comment by furushchev** [comment]
manually rebased #68

**Comment by furushchev** [comment]
@k-okada Please merge if there is no problem.

**Pull request merged**